diesel/src/expression/cast.rs

```diff
@@ -121,11 +122,15 @@ type_name! {
         Interval => "interval",
         Time => "time",
         Timestamp => "timestamp",
+        "uuid"; Uuid => "uuid",
```

Comment:
Is the Uuid **sql** type not defined unconditionally on whether the uuid feature is enabled? (If yes then cfg directives below should probably also be removed)

Reply:
I must misread the doc, thanks, I dropped the uuid feature